startup/index.js

```diff
@@ -17,6 +17,7 @@ const requireVars = [
   'SESSION_DOMAIN',
   'KEYSTONE_URL',
   'LAUNCHDARKLY_SDK_CLIENT_SIDE_ID',
+  'PERSONNEL_API_URL',
```

Comment:
Thought: Should we add this to the sysinfo handler in `src/pages/api/sysinfo.ts`? I think it might be helpful since we already have keystoneUrl there too.